app/src/main/java/com/internship/move/util/extensions.kt

```diff
@@ -0,0 +1,36 @@
+package com.internship.move.util
+
```

Comment:
We usually have an `extension` package inside `util` that has files like `TextViewExtensions.kt`
They tend to add up quickly and are hard to parse through after a while😅 That's why we try to keep them separate into Class specific extension files